jujugui/static/gui/src/app/state/state.js

```diff
@@ -98,6 +98,8 @@ const State = class State {
     */
     this._dispatchers = {};
 
+    this.sendAnalytics = cfg.sendAnalytics || null;
```

Comment:
What about `this.sendAnalytics = cfg.sendAnalytics || () = {};`
So that you can then avoid the condition below?
[shrug]